request_handlers_test.cc

```diff
@@ -36,10 +59,10 @@ TEST_F(RequestHandlersTest, EmptyStringTest) {
     "HTTP/1.1 200 OK\r\nContent-Type: text/plain\r\n\r\n";
   const size_t expected_response_buffer_size = header_size + request_buffer_size;
 
-  ASSERT_TRUE(HandleRequest(request_buffer,
-                            request_buffer_size,
-                            response_buffer,
-                            response_buffer_size));
+  ASSERT_TRUE(EchoHandleRequest(request_buffer,
```

Comment:
We definitely want to use the HandleRequest + RequestRouter interface. This is addressed by PR #43.